internal/verifier/change_stream.go

```diff
@@ -49,7 +52,7 @@ func (uee UnknownEventError) Error() string {
 
 // HandleChangeStreamEvent performs the necessary work for change stream events that occur during
 // operation.
-func (verifier *Verifier) HandleChangeStreamEvent(ctx context.Context, changeEvent *ParsedEvent) error {
+func (verifier *Verifier) HandleChangeStreamEvent(changeEvent *ParsedEvent) error {
```

Comment:
Just curious, what was the advantage of doing the buffering here versus closer to the read from the cursor? I was thinking we’d read all the events into a slice then pass that slice to this function (renamed `HandleChangeStreamEvents`).

Reply:
Good point, I made the changes.